peer/peer_test.go

```diff
@@ -200,12 +202,12 @@ func testPeer(t *testing.T, p *peer.Peer, s peerStats) {
 // TestPeerConnection tests connection between inbound and outbound peers.
 func TestPeerConnection(t *testing.T) {
 	verack := make(chan struct{})
```

Comment:
I would prefer that each peer will have its own channels, separate for input and output

Reply:
Why? These are just to make sure that handshake completed

Reply:
After a discussion we decided to fix this